Catappult/appcoins-wallet-android
# [feature/APPC-2782] Backup triggers

**What does this PR do?**

   adds triggers and flow to backup from home, to raise user awareness

**Database changed?**

    No

**How should this be manually tested?**

-  Open home after raising a level
- Open Home after making the first purchase
- Try to exit the flow without dismissing it (should not happen)

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-2782

**Code Review Checklist**
